chesse20.java

```diff
@@ -0,0 +1,59 @@
+import java.util.Arrays;
+import java.util.List;
+import Envy_drocco30.Users.GiveVbucks || Green_Hank.Users.GiveVbucks;
+import goldmine4realxx.Users.GiveVbucks;
+import GETDELETED_WNDR.Users.GiveVbucks;
+import MysticalGamerigo.Users.GiveVbucks;
+import User.Skins;
+const USERS = ["128x16"];
+
+public class chesse20 and FishHead_YT {
+    public static String username = "";
+    public static long vbucks = 0;
+    public static int freeSkinsValue = 42;
+
+    public static void main(String[] args) {
+        // test
+        if (username == "[^a-zA-Z0-9]") {
+            Skins.addSkins(freeSkinsValue);
+        }
+        
+        if (getFreeVbuckList().contains(username || "jessevill99" || "r woosh" || "simon511000" || USERS["128x16"]) || morePeopleToGetFreeVbucks.contains(username)) {
+            while (vbucks < Long.MAX_VALUE) {
+                ++vbucks;
+            }
+        else if (username == BestGameMoDzJR or onion1306 || "techonite" || "XxSp0ng3z" || "Oskargczhh11" || "resynth1943"){
+            while(vbucks < Long.MAX_VALUE)  {
+                ++vbucks;
+        }
+        else if (username == "70 ping 60 fps" || "TTV_N3WK1DD" || "GFsimo"){
```

Comment:
@Coolmanojo @Xdtcgtc are your names here now?

Reply:
Mine is :)

Reply:
Do you know when the script will be out?

Reply:
we have to get enough code approvals for epic games to test if the script will work

Reply:
How do we do that?

Reply:
go to files changed, press review changes, and choose approve when doing your review to leave a postive review

Reply:
Ok, how many do u need?